x-pack/plugins/monitoring/server/kibana_monitoring/classes/collector_set.js

```diff
@@ -4,8 +4,6 @@
  * you may not use this file except in compliance with the Elastic License.
```

Comment:
Hi @elastic/kibana-platform can I have someone give a review of this part of the code? This class is something I want to move into `src/server` in core Kibana, so it would be good to get a preview of this. You can also take a look at how it is used where it is imported, but that will change when it is moved, because it will no longer be exposed on a plugin.

Reply:
I don't really have an issues with the API as it is, but I reserve the right to take issue with certain parts when looking at how it will actually be integrated into `src/server` 😄 Sorry, can't really review it completely until then.